app/views/groups/_chat_content.html.erb

```diff
@@ -0,0 +1,29 @@
+
+  <div class="card-header">
+    <div class="d-flex bd-highlight">
+      <div class="img_cont">
+        <%= cl_image_tag(group.photo.key, width: 100, height: 100, class: "avatar-large") %>
+        <span class="online_icon"></span>
+      </div>
+      <div class="group_info">
+        <h2><%= group.name %></h2>
+        <% group.users.each do |user| %>
+          <span class="button-group-users"><%= user.nickname %></span>
+        <% end %>
+      </div>
+    </div>
+  </div>
+    <div class="card-body msg_card_body">
```

Comment:
Not the most important but by convention, css class are split with '-' not '_'

Reply:
Correct